managed-configurations.md

```diff
@@ -1,5 +1,7 @@
+- Azure -> Privileged Identity Management -> Policies and Rules
```

Comment:
We don't use arrrows on the managed configs page. Rather, we use sub-bullets to form a tree/hierarchical view

Reply:
Good catch